mozilla/foundation.mozilla.org
# single quote block working on Article and Publication pages

Closes #7057 


**Link to sample test page**: https://foundation-s-quote-bloc-i7tmuf.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/

**Steps to test**: 

1. Visit https://foundation-s-quote-bloc-i7tmuf.herokuapp.com/en/publication-page-with-child-article-pages/fixed-title-article-page/, and scroll down to view the quote component. 
2. If everything looks ok, testing is complete! 

Note: Please note that the wagtail CI test is going to fail as I have not yet added migrations. Once this PR is approved, I will pull the latest migrations and add mine to ensure that they are up to date. 

## Checklist


**Changes in Models:**
